src/BlockEntities/FurnaceEntity.cpp

```diff
@@ -169,6 +169,25 @@ bool cFurnaceEntity::ContinueCooking(void)
 	return m_IsCooking;
 }
 
+int cFurnaceEntity::GetReward(void)
+{
+	float RawReward = m_CurrentRecipe->Reward;
+	float Multiplier = GetOutputSlot().m_ItemCount;
+	if ((RawReward != 0.0) && (Multiplier != 0.0))
+	{
+		RawReward *= Multiplier;
+		int Reward = int(RawReward);
```

Comment:
Don't use C-style casts. Use our functions `FloorC` or `CeilC` instead, then you can even use `auto` for the type.